translations/fr/site.php

```diff
@@ -129,4 +129,20 @@
     "English" => "Français",
     "All forms" => "Tous les formulaires",
     "Send" => "Envoyer",
+
```

Comment:
I don't see the `search-txt` key here.

Reply:
Replace search-txt with {{ searchLabel }}.